rollup-tests.config.js

```diff
@@ -22,11 +27,25 @@ export default {
       // configuration for the `virtual` plugin above.
       preferBuiltins: false,
     }),
-    commonjs(),
+    commonjs({
+      include: 'node_modules/**',
```

Comment:
This matches our other projects. In future I'd like to filter the packages that are "allowed" to use CommonJS to just those that need it, as this can speed up build times.